### PR TITLE
SCP-1194 - Change JSON serialisation/deserialisation on Haskell side to use numbers in JSON

### PR DESCRIPTION
**Pull request by palas**
Update Haskell side to use numbers instead of strings for numbers.
Also change intermediate JavaScript representation to use BigNumber instead of strings.

### CODE REVIEW

File: marlowe-playground-client/src/Language/Javascript/MarloweJS.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import bignumber = require("bignumber.js")
 
+const MAX_SAFE_INTEGER = 9007199254740991;
+
```
Comment (nau) [inline marlowe-playground-client/src/Language/Javascript/MarloweJS.ts:4]: You can use `Number.MAX_SAFE_INTEGER` directly, I think

File: marlowe-playground-client/src/Language/Javascript/MarloweJS.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import bignumber = require("bignumber.js")
```
Comment (nau) [inline marlowe-playground-client/src/Language/Javascript/MarloweJS.ts:1]: I think we use `big-integer.js` library everywhere, which uses native BigInt whenever possible.

File: marlowe-playground-client/src/Language/Javascript/MarloweJS.ts (hunk 1 above)
Comment (palas) [reply marlowe-playground-client/src/Language/Javascript/MarloweJS.ts:4]: Oh, you are right, I thought I would need to change the imports, but it just works

File: marlowe-playground-client/src/Language/Javascript/MarloweJS.ts (hunk 2 above)
Comment (palas) [reply marlowe-playground-client/src/Language/Javascript/MarloweJS.ts:1]: Kris just changed this, [now it is `bignumber`](https://github.com/input-output-hk/plutus/pull/2293/commits/6d56881132d8d584166bfa1e1dfe5442e429ec06#diff-6f6d69b59f3c9cc3160955dfe0a51c71L17). It seems it has the advantage that it serialises to JSON without using strings for numbers, which is quite neat.

**Pull request merged**